package/lib/src/beamer_router_delegate.dart

```diff
@@ -406,6 +408,11 @@ class BeamerRouterDelegate<T extends BeamState> extends RouterDelegate<Uri>
   void clearBeamLocationHistory() =>
       _beamLocationHistory.removeRange(0, _beamLocationHistory.length - 1);
 
+  /// Change the strategy to use for handling browser URL to [PathUrlStrategy].
+  ///
+  /// [PathUrlStrategy] uses the browser URL's pathname to represent Beamer's route name.
+  void setPathUrlStrategy() => url_strategy.setPathUrlStrategy();
```

Comment:
I think it would be better if we put this method as a `static` function to `Beamer` class. Then the user can just call `Beamer.setPathUrlStrategy()` before `runApp(MyApp());` in main.
I tried, it works. Feel free to update your PR :slightly_smiling_face: 

Thanks for this!

Reply:
Yes, you're right, that would be better. 

I updated the PR :)